recipes/brev/recipe.yaml

```diff
@@ -0,0 +1,49 @@
+context:
+  name: brev
+  version: "0.6.310"
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://github.com/brevdev/brev-cli/archive/refs/tags/v${{ version }}.tar.gz
+  sha256: d7d8a58878c037372022f0c1d8fa58e15eb3d198498b521332d05def84cff351
+
+build:
+  number: 0
+  skip:
+    - win
+  script:
+    - go-licenses save --save_path ../library_licenses .
+    - go build -ldflags="-s -w -X github.com/brevdev/brev-cli/brev-cli.Version=${{ version }}" -v -o $PREFIX/bin/brev
+    # Install bash, zsh, and fish completions 
+    - mkdir -p $PREFIX/share/bash-completion/completions
+    - brev completion bash > $PREFIX/share/bash-completion/completions/brev
+    - mkdir -p $PREFIX/share/zsh/site-functions
+    - brev completion zsh > $PREFIX/share/zsh/site-functions/_brev
+    - mkdir -p $PREFIX/share/fish/vendor_completions.d
+    - brev completion fish > $PREFIX/share/fish/vendor_completions.d/brev.fish
+
+requirements:
+  build:
+    - ${{ stdlib("c") }}
+    - ${{ compiler('go-nocgo') }}
+    - go-licenses
+
+tests:
+  - script:
+      - brev --version
+about:
+  homepage: https://developer.nvidia.com/brev
+  summary: 'Brev CLI'
+  description: |
+    CLI tool for managing workspaces provided by brev.dev
+  license: MIT
+  license_file: LICENSE
+  documentation: https://docs.nvidia.com/brev/latest/getting-started.html
+  repository: https://github.com/brevdev/brev-cli
+
+extra:
+  recipe-maintainers:
+    - ruben-arts
```

Comment:
:wave: @theFong would you or any members of the NVIDIA open source team like to be added as co-maintainers of this feedstock? If you see this after it gets merged you can be easily added by PR. :+1: